src/components/Comments.jsx

```diff
@@ -15,6 +21,13 @@ export default function Comments() {
     });
   }, [comments]);
 
+  const deleteHandler = (comment_id) => {
+    setIsDeleting(true);
+    deleteComment(comment_id).then(() => {
+      setIsDeleting(false);
+    });
```

Comment:
This is working well - one thing that would be good to add is some sort of error message if the delete request fails for any reason (for example if the internet cuts out). At the moment when this happens, the user sees the message "Deleting message..." for a while.
You could use a catch block to implement this